### PR TITLE
KOGITO-5937 Jenkins PR: enable build-chain

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-5937

Related PRs:
- https://github.com/kiegroup/kogito-runtimes/pull/1618
- https://github.com/kiegroup/kogito-apps/pull/1046
- https://github.com/kiegroup/kogito-examples/pull/912

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins retest this

File: .github/workflows/ci-pr.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
       - name: Build Chain ${{ matrix.java-version }}
         id: build-chain
-        uses: kiegroup/github-action-build-chain@v2.6.2
+        uses: kiegroup/github-action-build-chain@v2.6.11
         with:
           definition-file: https://raw.githubusercontent.com/${GROUP}/kogito-pipelines/${BRANCH}/.ci/pull-request-config.yaml
       - name: Publish Test Report
```
Comment (danielezonca) [inline .github/workflows/ci-pr.yml:49]: Can you update the publish report step to use
```yaml
      - name: Publish Test Report
        if: ${{ always() }}
        uses: ScaCap/action-surefire-report@v1.0.10
        with:
          fail_on_test_failures: true
          fail_if_no_tests: false
          skip_publishing: true
          report_paths: '**/*-reports/TEST-*.xml'
```
?
I think it has not been updated by mistake 🤔

**Review by danielezonca — APPROVED** [review]
Approving with minor comment

File: .github/workflows/ci-pr.yml (hunk 1 above)
Comment (radtriste) [reply .github/workflows/ci-pr.yml:49]: done

**Review by Ginxo — APPROVED** [review]
LGTM

**Comment by radtriste** [comment]
seems like apps UI cannot be built ... not related to this PR ...

**Comment by radtriste** [comment]
updated the sonarcloud maven command ... (generate-resources -> validate)

jenkins retest this

**Pull request merged**